### PR TITLE
[stable/minio] Fix TLS can't be enabled

### PR DESCRIPTION
**Pull request by TreeKat71**
[fix] $scheme always "http"

The syntax is error, and it leads to $schema always be "http"

#### Is this a new chart
No

#### What this PR does / why we need it:
TLS can't be enabled successfully due to the go template in a wrong way.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Special notes for your reviewer:
It has been corrected in #21373 and being wrong again in #21503.
Please check it, cause I use the latest helm chart and can't enable tls successfully.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped 
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by briend** [comment]
@capnajaw you might have problems again after this has been merged, please check your SSL certs, etc.

File: stable/minio/templates/statefulset.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {{ $drivesPerNode := .Values.drivesPerNode | int }}
 {{ $scheme := "http" }}
 {{- if .Values.tls.enabled }}
-{{ $scheme := "https" }}
+{{ $scheme = "https" }}
```
Comment (hdifilwy) [inline stable/minio/templates/statefulset.yaml:7]: I got error on this change with helm v2.10

`Error: parse error in "minio/templates/statefulset.yaml": template: minio/templates/statefulset.yaml:7: unexpected "=" in operand`

Comment (TreeKat71) [reply stable/minio/templates/statefulset.yaml:7]: It seems like the go version, please refer to https://github.com/helm/helm/issues/4987
BTW, I use Helm 3

**Pull request merged**